src/lib/Persistence/Legacy/MigrateRichTextNamespaces/Gateway/DoctrineDatabase.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\FieldTypeRichText\Persistence\Legacy\MigrateRichTextNamespaces\Gateway;
+
+use Doctrine\DBAL\Connection;
+use Ibexa\Contracts\FieldTypeRichText\Persistence\Legacy\MigrateRichTextNamespaces\AbstractGateway;
+
+/**
+ * @internal
+ */
+final class DoctrineDatabase extends AbstractGateway
+{
+    private const TABLE_CONTENT_ATTRIBUTE = 'ezcontentobject_attribute';
+    private const COLUMN_DATA_TEXT = 'data_text';
+    private const FIELD_TYPE_IDENTIFIER = 'ezrichtext';
+
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function migrate(array $values): int
+    {
+        $qb = $this->connection->createQueryBuilder();
```

Comment:
Naming: rather use full name so there's no ambiguity.
```suggestion
        $queryBuilder = $this->connection->createQueryBuilder();
```